ext/standard/tests/serialize/bug49649_1.phpt

```diff
@@ -33,7 +33,12 @@ class Foo
 $class = unserialize(base64_decode($serialized));
 var_dump($class);
 ?>
---EXPECT--
+--EXPECTF--
+Deprecated: Creation of dynamic property Foo::$private is deprecated in %s on line %d
+
+Deprecated: Creation of dynamic property Foo::$public is deprecated in %s on line %d
+
+Deprecated: Creation of dynamic property Foo::$notThere is deprecated in %s on line %d
```

Comment:
These deprecation notices should not be emitted. This is not ending up creating a dynamic property - it writes to a **declared** property of a different visibility instead [ext/standard/var_unserializer.re](https://github.com/php/php-src/pull/9354/files#diff-22314cb41b89cd823c95f1073b63956a80c293a1bb4bcc95d5b05d2388f75e3d)

(`Bug #49649 (unserialize() doesn't handle changes in property visibility) - to protected`)

```php
php > echo var_representation(base64_decode('TzozOiJGb28iOjQ6e3M6MTI6IgBGb28AcHJpdmF0ZSI7aToxO3M6MTI6IgAqAHByb3RlY3RlZCI7aToyO3M6NjoicHVibGljIjtpOjM7czo4OiJub3RUaGVyZSI7czozOiJvbGQiO30'));
"O:3:\"Foo\":4:{s:12:\"\x00Foo\x00private\";i:1;s:12:\"\x00*\x00protected\";i:2;s:6:\"public\";i:3;s:8:\"notThere\";s:3:\"old\";}"
```

(using var_representation PECL for better null byte output)